packages/neutrino-preset-jest/package.json

```diff
@@ -19,6 +19,7 @@
     "babel-plugin-transform-es2015-modules-commonjs": "^6.22.0",
     "babel-preset-jest": "^18.0.0",
     "deepmerge": "^1.3.2",
+    "eslint-plugin-jest": "^19.0.1",
```

Comment:
Does this plugin version support the version of Jest we are using here? Jest 19 was recently released but we can't yet switch to it because it's a breaking change.